### PR TITLE
Update what-is-internet.md

### PR DESCRIPTION
**Pull request by armstrongl**
I made some minor edits, but I tried to keep the original voice of the author intact as much as possible. I hope you don't mind :)

#### What roadmap does this PR target?

- [ ] Code Change
- [ ] Frontend Roadmap
- [ ] Backend Roadmap
- [ ] DevOps Roadmap
- [ ] All Roadmaps
- [x] Guides

#### Please acknowledge the items listed below

- [ ] I have discussed this contribution and got a go-ahead in an issue before opening this pull request.
- [x] This is not a duplicate issue. I have searched and there is no existing issue for this.
- [x] I understand that these roadmaps are highly opinionated. The purpose is to not to include everything out there in these roadmaps but to have everything that is most relevant today comparing to the other options listed.
- [x] I have read the [contribution docs](../contributing) before opening this PR.

#### Enter the details about the contribution

- Minor grammatical edits
- Minor clarity edits

### CODE REVIEW

File: content/guides/what-is-internet.md [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
+The Internet is a global network of computers connected to each other which communicate through a standardized set of protocols. 
 
-In the video below Vint Cerf, one of the "fathers of the internet" explains the history of how the internet works and how no one person or organization is really in charge of it.  
+In the video below, Vint Cerf, one of the "fathers of the internet," explains the history of how the Internet works and how no one person or organization is really in charge of it.  
 
 <iframe width="100%" height="400" src="https://www.youtube.com/embed/Dxcc6ycZ73M" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 
-## Wires, Cables & Wifi
+## Wires, Cables, and Wi-Fi
```
Comment (naXa777) [inline content/guides/what-is-internet.md:20]: why not ampersand? I like this symbol...
```suggestion
## Wires, Cables & Wi-Fi
```

**Review by naXa777 — APPROVED** [review]
generally I approve, but I pray you: don't remove **&**!

**Review by kamranahmedse — APPROVED** [review]
Thanks @armstrongl

**Review by SidBlack — COMMENTED** [review]
ok

**Pull request merged**